### PR TITLE
Tooltip added to each skill 

### LINKED ISSUE #16: Skills should have names
In the skill section, Non-technical individuals(like HR) can't predict the name of the skill by just seeing the icon. 
So, each skill should have a tooltip/title to show its name like "Tailwind CSS" .

Screenshot 2023-04-20 134340

### PR DESCRIPTION
**Pull request by sagar-io**
I've added the **tooltip to each skill with a hover effect** in the skills section of the portfolio

Without update: 

Screenshot 2023-04-20 152908

After update: 

Screenshot 2023-04-20 152955
Resolves #16

### CODE REVIEW

File: src/components/skills.jsx [hunk 1]
```diff
@@ -3,8 +3,9 @@ import React from "react";
 function Skills({ image, name }) {
   return (
     <section>
-      <div className="lg:max-w-sm p-6  flex justify-center ">
-        <img src={image} alt={name} className="h-10" />
+      <div className="lg:max-w-sm p-6  flex justify-center relative">
+        <img src={image} alt={name} className="peer h-10 cursor-none" />
```
Comment (priyankarpal) [inline src/components/skills.jsx:7]: remove `cursor-none`
```suggestion
        <img src={image} alt={name} className="peer h-10 " />
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

**Comment by sagar-io** [comment]
I removed the "cursor-none" class.

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Comment by sagar-io** [comment]
> Looks Good 🎉

Yeah, thanks : )

**Pull request merged**